compiler/ast/ndi.nim

```diff
@@ -24,24 +25,38 @@ import
   ]
 
 type
+  Mapping = tuple
+    info: TLineInfo ## the source code position where the symbol is defined
+    orig: PIdent ## the user-defined name
+    name: string ## the mangled name
```

Comment:
Would it be worth storing the symbol I'd as a last ditch but of information for tracing issues?

Reply:
Hm, I'm not sure. I think issues in this area are unlikely, but storing an additional ID here is rather cheap, so I'm going to add it.